src/components/js/EcAppCard.js

```diff
@@ -17,6 +18,11 @@ export default {
     descriptionMaxLength: {
       type: Number,
       default: 75
+    },
+
+    newVersionAvailable: {
+      type: Boolean,
+      default: false
     }
```

Comment:
Quando definindo API de componente, é melhor que booleanos tenham prefixo `is`, `has` ou `can`